webpage/src/components/Table.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+
+const join = (array, separator) =>
+  array.reduce((result, child, index) => {
+    if (index < array.length - 1) {
+      return result.concat(child, separator);
+    } else {
+      return result.concat(child);
+    }
+  }, []);
+
+const symbol = value => {
+  if (value === true) {
+    return "✓";
+  } else if (value === false) {
+    return "";
+  } else {
+    return "?";
+  }
+};
+
+const CellTd = options => {
+  const { header, value } = options;
+  if (header === "Package") {
```

Comment:
This recurring pattern (if Package else if Version) doesn't seem right.
Do you mind implementing another solution?
Like using a map, or different components.

Reply:
```js
const fields = {
    "Package": value => { ... return join(links, " + "); },
    "Version": x => x,
    "default": symbol
}
const field = fields[header] || fields["default"]
return <td>{ field(value) }</td>
```